src/library/builtin.lisp

```diff
@@ -17,6 +17,8 @@
   (declare fix (((:a -> :b) -> (:a -> :b)) -> (:a -> :b)))
   (define (fix f n)
     "The factorial function can be written
```

Comment:
I think this function should have better documentation.

Reply:
I don't really know how to describe it. And I don't want to copy the description from Haskell because I think the semantics might be a little different w/ strictness. If you have a better docstring I can add it.

Reply:
```
Compute the fixed point of a unary function. This is equivalent to the Y-combinator of the lambda calculus.

This combinator allows recursion without specific assignment of names. For example, ..."
```